events/api.py

```diff
@@ -529,6 +529,23 @@ def create(self, validated_data):
                 raise error
         return instance
 
+    def create_keyword(self, validated_data):
+        if 'data_source' not in validated_data:
+            validated_data['data_source'] = self.data_source
+        # no django user exists for the api key
+        if isinstance(self.user, ApiKeyUser):
+            self.user = None
+        validated_data['created_by'] = self.user
+        validated_data['last_modified_by'] = self.user
+        try:
+            instance = super().create(validated_data)
```

Comment:
Just `return super()...` instead of storing into a local you immediately return.